src/AppInstallerCLITests/RestInterface_1_0.cpp

```diff
@@ -0,0 +1,413 @@
+                    "License": "Foo Bar License",
+                    "LicenseUrl": "http://licenseUrl-fr.net",
+                    "Copyright": "Foo Bar Copyright",
+                    "CopyrightUrl": "http://copyrightUrl-fr.net",
+                    "ShortDescription": "Foo bar is a foo bar French.",
+                    "Description": "Foo bar is a placeholder French.",
+                    "Tags": [
+                      "FooBarFr",
+                      "FooFr",
+                      "BarFr"
+                    ]
+                  }
+                ],
+                "Installers": [
+                  {
+                    "InstallerSha256": "011048877dfaef109801b3f3ab2b60afc74f3fc4f7b3430e0c897f5da1df84b6",
+                    "InstallerUrl": "http://foobar.exe",
+                    "Architecture": "x86",
+                    "InstallerLocale": "en-US",
+                    "Platform": [
+                      "Windows.Desktop"
+                    ],
+                    "MinimumOSVersion": "1078",
+                    "InstallerType": "msix",
+                    "Scope": "user",
+                    "SignatureSha256": "011048877dfaef109801b3f3ab2b60afc74f3fc4f7b3430e0c897f5da1df84b6",
+                    "InstallModes": [
+                      "interactive"
+                    ],
+                    "InstallerSwitches": {
+                      "Silent": "/s",
+                      "SilentWithProgress": "/s",
+                      "Interactive": "/i",
+                      "InstallLocation": "C:\\Users\\User1",
+                      "Log": "/l",
+                      "Upgrade": "/u",
+                      "Custom": "/custom"
+                    },
+                    "InstallerSuccessCodes": [
+                      0
+                    ],
+                    "UpgradeBehavior": "install",
+                    "Commands": [
+                      "command1"
+                    ],
+                    "Protocols": [
+                       "protocol1"
+                    ],
+                    "FileExtensions": [
+                      ".fileextension"
+                    ],
+                    "Dependencies": {
+                      "WindowsFeatures": [
+                        "feature1"
+                      ],
+                      "WindowsLibraries": [
+                        "library1"
+                      ],
+                      "PackageDependencies": [
+                        {
+                          "PackageIdentifier": "Foo.Baz",
+                          "MinimumVersion": "2.0.0"
+                        }
+                      ],
+                      "ExternalDependencies": [
+                        "FooBarBaz"
+                      ]
+                    },
+                    "PackageFamilyName": "FooBar.PackageFamilyName",
+                    "ProductCode": "",
+                    "Capabilities": [
+                      "bluetooth"

```

Comment:
Sigh, Bluetooth is a brand, it really should never be written in lowercase. But, I'm assuming this is an API of some sort.